vacationtime-solution/src/Code.js

```diff
@@ -0,0 +1,293 @@
+ * Creates a new column for a manager to input her approval
+ * of each employee's vacation request. Uses a helper function
+ * to create an additional notification column.
+ */
+function createNewColumns() {
+  var sheet = SpreadsheetApp.getActiveSheet();
+  var lastCol = sheet.getLastColumn();
+  var lastRow = sheet.getLastRow();
+  var frozenRows = sheet.getFrozenRows();
+  var startRow = frozenRows + 1;
+  var numRows = lastRow - frozenRows;
+
+  // Creates approval column.
+  sheet.insertColumnAfter(lastCol);
+  sheet.getRange(frozenRows, COLUMN_NUMBER.APPROVAL)
+      .setValue('APPROVAL');
+
+  // Sets drop-down menu cells in approval column.
+  var approvalColumnRange = sheet.getRange(startRow, COLUMN_NUMBER.APPROVAL,
+      numRows, 1);
+  var dropdownValues = ['APPROVED', 'NOT APPROVED', 'IN PROGRESS'];
+  var rule = SpreadsheetApp.newDataValidation().requireValueInList(dropdownValues)
+      .build();
+  approvalColumnRange.setDataValidation(rule);
+  approvalColumnRange.setValue('IN PROGRESS');
+  
+  // Calls helper function to repeat the above code but for the NOTIFIED column.
+  createNotifiedColumn();
+}
+
+/** 
+ * Adds a column to allow managers to view which employees
+ * have or have not yet been notified. The value of the cells
+ * is set to 'NOT NOTIFIED' on default & changed accordingly.
+ */
+function createNotifiedColumn() {
+  var sheet = SpreadsheetApp.getActiveSheet();
+  var lastCol = sheet.getLastColumn();
+  var lastRow = sheet.getLastRow();
+  var frozenRows = sheet.getFrozenRows();
+  var startRow = frozenRows + 1;
+  var numRows = lastRow - frozenRows;
+  
+  // Sets up column properties.
+  sheet.insertColumnAfter(lastCol); 
+  sheet.getRange(frozenRows, COLUMN_NUMBER.NOTIFIED)
+      .setValue('NOTIFIED STATUS');
+
+  // Sets column's cells to be drop-down menus.
+  var notifiedColumnRange = sheet.getRange(startRow, COLUMN_NUMBER.NOTIFIED, 
+                                           numRows, 1);
+  var dropdownValues = ['NOTIFIED', 'NOT NOTIFIED'];
+  var rule = SpreadsheetApp.newDataValidation().requireValueInList(dropdownValues)
+      .build();
+  notifiedColumnRange.setDataValidation(rule);
+  notifiedColumnRange.setValue('NOT NOTIFIED');
+}
+
+/**
+ * Creates a calendar event for an employee whose vacation
+ * request has been approved.
+ * @param {String} employeeName Name of employee.
+ * @param {String} employeeEmail Email of employee.
+ * @param {date} startDate Vacation request start date.
+ * @param {date} endDate Vacation request end date.
+**/
+function createCalEvent(employeeName, employeeEmail,
+                       startDate, endDate) { 
+    var managerCal = CalendarApp.getCalendarById(MANAGER_EMAIL);
+    // Creates a calendar event.
+    var event = managerCal.createEvent(EVENT_TITLE + employeeName,
+                                       startDate, endDate, {
+        description: 'Your vacation time from ' +
+                                            startDate + ' to ' + endDate +
```

Comment:
Also, might want to consider using Utilities.stringFormat() instead of concatenating here. More of a style than functional issue, just tend to find it slightly more readable for cases like this. Applies to sendRejectionEmail too.